pkg/features/kube_features.go

```diff
@@ -837,7 +838,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	CSIMigrationvSphere: {Default: true, PreRelease: featuregate.Beta},
 
-	CSINodeExpandSecret: {Default: false, PreRelease: featuregate.Alpha},
+	CSINodeExpandSecret: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
The [Feature] tag should be removed from the e2e test so that it runs in normal CI jobs. 

However, it also depends on https://github.com/kubernetes/kubernetes/issues/116515 to be fixed.

Reply:
ack.. thanks @msau42 and @pacoxu ..